deploy/compose/example/local/README.md

```
@@ -0,0 +1,70 @@
+
+```bash
+mkcert -install
+mkdir -p certs
+mkcert -cert-file certs/tls.crt -key-file certs/tls.key localhost 127.0.0.1 ::1
+```
+
+Update `.env`:
+
+```bash
+EXT_ENDPOINT=https://localhost
+TLS_CONF=./config/nginx/tls.conf
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)